src/utils/notifications.js

```diff
@@ -179,3 +182,14 @@ export const processNotification = (notification: Object, myEthAddress?: string)
 
   return result;
 };
+
+export const parseNotificationMessage = (notification: Object): Object => {
+  const createdAt = get(notification, 'createdAt');
```

Comment:
Do we really need the `get` here?

Not sure if we expect  the parameter `notification` to be `null`/`undefined`. In that case, we should mark it as so. But then, if `notification` is `null`/`undefined` we can avoid the rest of the flow. Maybe we can make sure the caller passes in something instead of controlling the argument here (or else, just add an `if (!notification) return ...;` at the top of the function).

```js
const mappedInviteNotifications = inviteNotifications
  .filter(notification => !!notification)
  .map(parseNotificationMessage)
  // ...
```

Also, typing the parameter will make flow much helpful on letting us know what to check for:

```js
type RawNotificationMessage = {
  createdAt?: number, // number, I guess
  payload?: {
    msg?: string,
  }
};

export const parseNotificationMessage = (
  notification?: RawNotificationMessage,
): Object => {
  // Controlling the function flow with an argument
  // is normally something to be avoided, but if we
  // really want to make this optional:
  if (!notification) {
    return {};
  }

  const { createdAt, payload = {} } = notification;
  if (payload.msg) {
    try {
      const parsedMessage = JSON.parse(payload.msg);

      return { ...parsedMessage, createdAt };
    } catch (e) {
      //
    }
  }
  return { createdAt };
};
```

Reply:
I've added notification object as TODO, but will now go through back-end and will try to create a single model, thanks 👍